google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1alpha2/StreamWriter.java

```diff
@@ -183,6 +192,27 @@ public String getTableNameString() {
     return tableName;
   }
 
+  /* Once updated schema is retrieved, it will be set to null (matches backend implementation of clearing updated schema once it puts it in an AppendRowsResponse)
+   *
+   * @return Table.TableSchema
+   */
+  public Table.TableSchema getUpdatedSchema() {
```

Comment:
Do we need this function?

Reply:
This was to pass out the schema to the JsonStreamWriter, and technically it's not needed, just thought of letting the users see the schema. I'll remove them then.